Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rsrchboy/perl-v5.16
+MAINTAINER Xinjiang Shao <shaoxinjiang@gmail.com>
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    git \
+    libssl-dev \
+    libmpfr-dev \
+    nodejs \
+    npm \
+    nodejs-legacy \
+    && rm -fr /var/lib/apt/lists/*
+RUN npm install -g handlebars@1.3.0 uglifyjs
```

Comment:
@moollaza 

It would look similar with the following error message.

```console
root@c4a5f0437d95:/home/ddg# dzil authordeps --missing
root@c4a5f0437d95:/home/ddg# node -v
bash: node: command not found
root@c4a5f0437d95:/home/ddg# dzil listdeps
->subdir called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7; downstream code should be updated to use Path::Tiny API, not Path::Class at /usr/local/perlbrew/perls/5.16/lib/site_perl/5.16.3/Dist/Zilla/Plugin/AutoModuleShareDirs.pm line 57.
[BuildShareAssets] FATAL - Failed to run handlebars: -1
[BuildShareAssets] FATAL - Failed to run handlebars: -1 at /usr/local/perlbrew/perls/5.16/lib/site_perl/5.16.3/x86_64-linux/Moose/Meta/Method/Delegation.pm line 110.
```
``Path::Tiny API `` warnings would be fixed after the package is updated, but I cannot get rid of Fatal error from BuildShareAssets if I don't install node